app/src/main/java/com/example/csc207courseproject/entities/TournamentSets.java

```diff
@@ -0,0 +1,13 @@
+package com.example.csc207courseproject.entities;
+
```

Comment:
I don't think this class is necessary as you can just pull ongoing sets into the state from the DAO